src/common_interface/integrator_types.jl

```diff
@@ -147,6 +147,10 @@ DiffEqBase.check_error(integrator::AbstractSundialsIntegrator) =
     interpret_sundials_retcode(integrator.flag)
 
 DiffEqBase.postamble!(integrator::AbstractSundialsIntegrator) = nothing
+
+# false because this function is used in callbacks and cannot change dt forcibly
+DiffEqBase.is_integrator_adaptive(integrator::AbstractSundialsIntegrator) = false
```

Comment:
requires a DiffEqBase version

Reply:
I guess it doesn't technically error before that though.

Reply:
I got a message from one of the users about the error. I used `integrator.opts.adaptive` directly in apply callback

Reply:
Though it's not required with the isdefined now